### PR TITLE
Positive feedback

### PR DESCRIPTION
**Pull request by przadka**

### CODE REVIEW

File: agreements.md [hunk 1]
```diff
@@ -176,6 +176,32 @@ This solution is meant to allow the team to auto-calibrate and recognize what co
...
+- Person B: “I don't understand. Could you please rephrase your concern in a more constructive way?”
+- Person A: “Sorry. I am worried this design has some vulnerabilities that someone could exploit. How can we make sure that this does not happen?”
+
+### Example 2
+
+- Person A: “This is obvious. Anybody above the age of 6 knows this!”
+- Person C: “Hey, Person B put some work into this. Could you rephrase your feedback in a more constructive way?”
+- Person A: “Sorry. My concern here is that we should write only things that are not obvious and not generic. Otherwise, this won't provide much value.”
+
+This method allows us to quickly adjust our communication and ensure feedback is always delivered constructively.
```
Comment (mjurbanski-reef) [inline agreements.md:203]:
```suggestion
```

this was just a repeat of the same thing

File: agreements.md [hunk 2]
```diff
@@ -176,6 +176,32 @@ This solution is meant to allow the team to auto-calibrate and recognize what co
 If the offensive comment is seen as severe and above the grade of this agreement, it will be raised to the HR circle to decide on further steps.
 Fortunately, so far, there have been no such situations.
 
+## Constructive feedback
+
+We aim to maintain a positive work environment where feedback is given constructively to promote growth and improvement.
```
Comment (mjurbanski-reef) [inline agreements.md:181]:
```suggestion
We aim to maintain a positive work environment where feedback is given constructively to promote growth.
```
growth and improvement seem to be exactly same thing in this context

File: agreements.md [hunk 3]
```diff
@@ -176,6 +176,32 @@ This solution is meant to allow the team to auto-calibrate and recognize what co
...
+To manage situations where feedback might come across as negative or aggressive, we will use the following procedure:
+
+- If someone feels that feedback is too negative or aggressive towards any staff member, they should say:
```
Comment (mjurbanski-reef) [inline agreements.md:184]:
```suggestion
To manage situations where feedback may seem nonconstructive, we will use the following procedure:

- If someone feels that feedback is overly negative or aggressive towards any staff member, they should say:
```

negative feedback can be just constructive, rephrased a little to make it clearer we are concerned about non-constructive criticism

File: agreements.md [hunk 4]
```diff
@@ -176,6 +176,32 @@ This solution is meant to allow the team to auto-calibrate and recognize what co
...
+  “Could you please rephrase your concern in a more constructive way?”
+- The person who provided the feedback should then respond with:
+  “Sorry” and rephrase their feedback in a more constructive manner.
+- No further comments or defenses should be made by others present.
```
Comment (mjurbanski-reef) [inline agreements.md:188]:
```suggestion
```

this works for joke agreement, but doesn't work in what is supposed to be a discussion.
see example 1 - it ends with a question, so obviously more comments will follow

File: agreements.md (hunk 2 above)
Comment (przadka) [reply agreements.md:181]: growth = personal, improvement = process

if we want to keep one I would keep improvement. @mjurbanski-reef what do you think?

**Review by mjurbanski-reef — APPROVED** [review]

**Pull request merged**